### PR TITLE
Layouts

### PR DESCRIPTION
**Pull request by mikbry**

### CODE REVIEW

File: src/libs/components/list/listDragComponent.jsx [hunk 1]
```diff
@@ -15,10 +15,10 @@ import ListDragItem from "./listDragItem";
 class ListDragComponent extends Component {
   render() {
     const {
-      items, selectedItem, onSelect, onMove, onDrop, className,
+      items, selectedItem, onSelect, onMove, onDrop, ...otherProps
     } = this.props;
     return (
-      <List className={className}>
+      <List {...otherProps} >
```
Comment (willdurand) [inline src/libs/components/list/listDragComponent.jsx]: I am not a huge fan of spreading props like this because you will likely spread invalid props.

Comment (mikbry) [reply src/libs/components/list/listDragComponent.jsx]: List will validate them. I checked React docs and that the way they do.

**Comment by willdurand** [comment]
🤘

**Comment by mikbry** [comment]
What's the meaning of this square ? :-)

**Comment by willdurand** [comment]
screen shot 2018-03-23 at 18 16 48

**Pull request merged**